.github/workflows/release.yml

```
@@ -1,9 +1,16 @@
 name: release
 
 on:
-  push:
-    tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
+  workflow_dispatch:
+    inputs:
+      versionNumber:
+        description: 'Release version number'
```

Should this let folks know that it should include the `v` prefix?

```suggestion
        description: 'Release version number (v#.#.#)'
```